src/bufr/DataObject.h

```diff
@@ -657,6 +667,24 @@ namespace Ingester
             return 0.0f;
         }
 
+        /// \brief Get the data at the index as a int for numeric data.
+        /// \return Int data.
```

Comment:
Comments should be updated: they refer to integer type but these functions are for strings.